src/app/main/component/ubs/components/ubs-personal-information/ubs-personal-information.component.html

```diff
@@ -104,19 +110,24 @@ <h5 class="comment-title">{{ 'personal-info.info-address-coment' | translate }}<
       <app-ubs-input-error [formElement]="getControl('addressComment')" [controlName]="'addressComment'"> </app-ubs-input-error>
     </div>
   </div>
-  <div class="w-100 d-flex justify-content-between personal-data-navigation">
-    <button class="secondary-global-button btn" matStepperPrevious>{{ 'personal-info.info-back' | translate }}</button>
-    <div>
-      <button class="secondary-global-button btn" type="button" (click)="cancelUBS()">{{ 'personal-info.info-cancel' | translate }}</button>
-      <button
-        type="submit"
-        (click)="submit()"
-        class="primary-global-button btn"
-        matStepperNext
-        [disabled]="personalDataForm.invalid || !addresses?.length"
-      >
-        {{ 'personal-info.info-next' | translate }}
-      </button>
-    </div>
-  </div>
 </form>
+
+<div class="w-100 d-flex justify-content-between buttons">
+  <button class="secondary-global-button btn back-btn m-0" matStepperPrevious>
+    <i class="fas fa-chevron-left"></i> &nbsp; {{ 'personal-info.info-back' | translate }}
+  </button>
+  <div class="w-100 d-flex justify-content-end">
+    <button class="secondary-global-button btn cansel-button" type="button" (click)="cancelUBS()">
+      {{ 'personal-info.info-cancel' | translate }}
+    </button>
+    <button
+      type="submit"
+      (click)="submit()"
+      class="primary-global-button btn m-0"
+      matStepperNext
+      [disabled]="personalDataForm.invalid || !addresses?.length"
```

Comment:
not sure this going to work here..
how about to move this to ts file ?

Reply:
'!addresses?.length' - this?

Reply:
yes

Reply:
Yea, it was useless there and I removed it. Everything works fine